src/commands/mods-configure.ts

```diff
@@ -12,6 +12,7 @@ import { logPrefix } from "../mods/modsHelper";
 import * as paramHelper from "../mods/paramHelper";
 import * as requirePermissions from "../requirePermissions";
 import * as utils from "../utils";
+import { logger } from "..";
```

Comment:
Why are you using the `logger` from the `client` rather than importing `logger` directly yourself?

Reply:
Fixed - i think this autoimported